e2e/playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig } from '@playwright/test';
+import { AzureReporterOptions } from '@alex_neo/playwright-azure-reporter/dist/playwright-azure-reporter';
+import path from 'path';
+const envPath = path.resolve(__dirname, '.env');
+import dotenv from 'dotenv';
+dotenv.config({ path: envPath });
+
+export default defineConfig({
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: 2,
+  /* Opt out of parallel tests on CI. */
+  /* Reporter to use. See https://playwright.devdocs/test-reporters */
+  reporter: [
+    ['list'],
+    [
+      '@alex_neo/playwright-azure-reporter',
+      {
+        orgUrl: 'https://dev.azure.com/redcrossnl',
+        token: process.env.AZURE_DEVOPS_TOKEN,
+        planId: 27408,
+        projectName: '121 Platform',
+        environment: 'AQA',
+        logging: true,
+        testRunTitle: 'Playwright Test Run',
+        publishTestResultsMode: 'testRun',
+        uploadAttachments: true,
+        attachmentsType: ['screenshot', 'video', 'trace'],
+        testRunConfig: {
+          owner: {
+            displayName: 'Krajewski, Piotr',
```

Comment:
Wouldn't this value need to be dynamic to be able to run these tests "as someone else"?

Or could we just leave any value out (and assume the "default owner" is good enough)?

Reply:
I think for now the test suite is under `Krajewski, Piotr` name and it validated against token with his name during test run. Thats why he keeps his name. We could create a generic token later with name `510 Redcross` or similar and use that.